scavenger-api/build.gradle.kts

```diff
@@ -3,9 +3,9 @@ import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 import org.springframework.boot.gradle.tasks.bundling.BootJar
 
 plugins {
-    val kotlinVersion = "1.8.10"
-    val springBootVersion = "2.5.12"
-    val springDependencyManagementVersion = "1.0.11.RELEASE"
+    val kotlinVersion = "1.9.0"
```

Comment:
Using the `1.9.24` version

Reply:
fixed in https://github.com/naver/scavenger/pull/124/commits/576081e870da04e39edc820836090269dcec0a4a